tests/ui-internal/author.rs

```diff
@@ -1,3 +1,5 @@
+#![warn(clippy::author)]
+
```

Comment:
@blyxyas Can you shed some light on why the `author` lint tests were moved to `ui-internal` and this `warn` was added? (This lint never emits a diagnostic, so `warn` on it doesn't make sense)